deployment/build-s3-dist.sh

```diff
@@ -148,9 +148,10 @@ cat requirements.txt.old | grep -v "Media_Insights_Engine_Lambda_Helper" > requi
 echo "/packages/$file" >> requirements.txt;
 # Build Lambda layer zip files and rename them to the filenames expected by media-insights-stack.yaml. The Lambda layer build script runs in Docker.
 # If Docker is not installed, then we'll use prebuilt Lambda layer zip files.
-echo "Running build-lambda-layer.sh"
+echo "Running build-lambda-layer.sh:"
+echo ""
 rm -rf lambda_layer-python-* lambda_layer-python*.zip
-./build-lambda-layer.sh requirements.txt > /dev/null
+./build-lambda-layer.sh requirements.txt
```

Comment:
I think the redirect to /dev/null is required for the conditional check below. Maybe @ianwow can chime in.

Reply:
I put the redirect to /dev/null there in order to hide the compilation details for MediaInfo, because during the compilation there is an innocuous error which makes it look like the MIE build is failing, when in fact it was working fine.

I recommend leaving this redirect as long as we're compiling MediaInfo from source.

Reply:
I added the redirect back